pkgs/applications/misc/udiskie/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, asciidoc-full, gettext
+, gobjectIntrospection, gtk3, hicolor_icon_theme, libnotify
+, pythonPackages, udisks2, wrapGAppsHook }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "udiskie-${version}";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "coldfix";
+    repo = "udiskie";
+    rev = version;
+    sha256 = "01x5fvllb262x6r3547l23z7p6hr7ddz034bkhmj2cqmf83sxwxd";
+  };
+
+  preConfigure = ''
+    export XDG_RUNTIME_DIR=/tmp
```

Comment:
@bjornfor could you explain why you think it is weird or wrong? I can imagine it is strange to set something outside of a phase, and so it might not be clear when exactly these env vars are set.